pkg/asset/manifests/ingress.go

```diff
@@ -2,6 +2,7 @@ package manifests
 
 import (
 	"fmt"
+	"github.com/openshift/installer/pkg/types/aws"
```

Comment:
Nit: this line should be in the installer imports block.

Reply:
Done